app/env_cmd.go

```diff
@@ -23,6 +23,8 @@ type envCmd struct {
 	Unset             bool   `xor:"action" short:"u" help:"Unset the specified environment variable."`
 	Name              string `arg:"" optional:"" help:"Name of the environment variable."`
 	Value             string `arg:"" optional:"" help:"Value to set the variable to."`
+
+	HermitBin string `help:"The location of the hermit binary." env:"HERMIT_ROOT_BIN" default:"$HOME/bin/hermit"`
```

Comment:
This is essentially a global setting so it should be added to `cliInterface` as a method, and the flag added to `cliBase`. Then in each of these Run methods inject it eg. `Run(cli cliInterface)`